mobile-app/app/screens/AppNavigator/screens/Loans/screens/ConfirmCreateVaultScreen.tsx

```diff
@@ -127,7 +161,7 @@ function SummaryTransactionDetails (props: {fee: BigNumber}): JSX.Element {
       />
       <FeeInfoRow
         type='VAULT_FEE'
-        value={vaultFee.toFixed(8)}
+        value={vaultFee.toFixed(0)}
```

Comment:
I think we should maintain the 8 decimal places, for consistency sake that we display 8 dp for all amount related to token

Reply:
Ok will change, I was actually thinking about this since it's always (2) but I guess for sake of consistency